workshops/dodge/README.md

```diff
@@ -286,9 +315,9 @@ Et voilà! Our enemy now shows up randomly across the top of the canvas! And dro
 
 Well, if you've failed to dodge the enemy at least once, you'll realize that the player is not forced to frantically dodge after all... the enemy doesn't attack it at all.
 
-Let's make it a real threat by recognizing when the enemy and player collide. P5play provides the `overlap()` method on our sprites for doing just that.
+Let's make it a real threat by recognizing when the enemy and player collide. P5.Play provides the `overlap()` method on our sprites for doing just that.
```

Comment:
Yeah, gotta fix that.

On Wed, Apr 6, 2016 at 10:38 AM, JESSICA <notifications@github.com> wrote:

> In workshops/dodge/README.md
> <https://github.com/jkwok91/hackclub/pull/2#discussion_r58748739>:
>
> > @@ -286,9 +315,9 @@ Et voilà! Our enemy now shows up randomly across the top of the canvas! And dro
> >
> >  Well, if you've failed to dodge the enemy at least once, you'll realize that the player is not forced to frantically dodge after all... the enemy doesn't attack it at all.
> >
> > -Let's make it a real threat by recognizing when the enemy and player collide. P5play provides the `overlap()` method on our sprites for doing just that.
> > +Let's make it a real threat by recognizing when the enemy and player collide. P5.Play provides the `overlap()` method on our sprites for doing just that.
>
> weird caps
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/jkwok91/hackclub/pull/2/files/f24ad8246cdf7e3e53e56b84653a167ecb9fe63a#r58748739>
>



-- 
Zach Latta
https://zachlatta.com <http://zachlatta.com>